recipes-ni/ni-utils/fw-printenv_1.0.bb

```diff
@@ -1,37 +1,41 @@
-SUMMARY = "fw_printenv/fw_setenv utility for x64 systems"
-DESCRIPTION = "distro-specific fw_printenv utility for reading firmware variables."
+SUMMARY = "fw_printenv/fw_setenv utility"
+DESCRIPTION = "NI-LinuxRT-specific fw_printenv utility for reading firmware variables."
+HOMEPAGE = "https://github.com/ni/meta-nilrt"
+SECTION = "base"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
-SECTION = "base"
+
+COMPATIBLE_MACHINE = "x64"
+DEPENDS = "niacctbase"
 
 SRC_URI = "\
-	file://fw_printenv \
 	file://EFI_NI_vars \
 	file://SMBIOS_NI_vars \
+	file://fw_printenv \
 	file://grubvar_readonly \
 "
 
-COMPATIBLE_MACHINE = "x64"
-
-FILES_${PN} += "${datadir}/fw_printenv/*"
-
-DEPENDS += "shadow-native pseudo-native niacctbase"
```

Comment:
Looks like we lost a couple dependencies here.

Reply:
`shadow-native` and `pseudo-native` aren't needed for recipes which only install content under uid 0 - like this one. The deps were added to this recipe - and many others - in a bulk commit from years ago, which wasn't conceptually correct.